File: MY C PROGRAMS/CIRCLE.cpp
//PROGRAM TO FIND AREA AND CIRCUMFERENCE OF A CIRCLE.

#include<stdio.h>

float calarea(float rad);
float calcircum(float rad);
int main()
{
	float r,area,circum;
	printf("Enter the radius: ");
	scanf("%f",&r);
	area=calarea(r);
	circum=calcircum(r);
	printf("Area of circle=%.2f\n",area);
   	printf("circumference of circle=%.2f\n",circum);
	   return 0;
	   }
	   
	   float calarea(float rad)
	   {
	   float a=3.14*rad*rad;
	   	return (a);
	   }
      float calcircum(float rad)
      {
      float c=2*3.14;rad;
	    return (c);
	  }
	  

File: PYTHON CLASS/circle.py
from math import pi
def circle(r):
    peri=2*pi*r 
    area=pi*r*r
    return(peri,area)
radius=int(input("Enter the radius of circle"))
(peri,area)=circle(radius)
print(f"Perimeter={peri:.2f},Area={area:.2f}")